src/luasandbox_defines.h

```diff
@@ -10,7 +10,9 @@
 #define luasandbox_defines_h_
 
 #ifdef _WIN32
+#if _MSC_VER < 1900
 #define snprintf _snprintf
+#endif // 
```

Comment:
Did you intend to add an  __MSC_VER close comment?

Also the test header uses the same define
https://github.com/mozilla-services/lua_sandbox/blob/b211bc427e49fa92e54d679996fe2ba06542ba1b/include/luasandbox/test/mu_test.h#L15